webview/package.json

```diff
@@ -72,6 +73,7 @@
     "raw-loader": "4.0.2",
     "sass": "1.62.0",
     "sass-loader": "13.2.2",
+    "storybook": "7.0.7",
     "storybook-addon-designs": "6.3.1",
```

Comment:
```
Attention: We've detected that you're using the following addons in versions which are known to be incompatible with Storybook 7:

- storybook-addon-designs@6.3.1

Please be aware they might not work in Storybook 7. Reach out to their maintainers for updates and check the following Github issue for more information:
https://github.com/storybookjs/storybook/issues/20529
```

Got this warning when I ran storybook. We can probably just remove the addon. We're only using it for one storybook and it's not working since the Figma URL is outdated, I believe.

Reply:
Done

Reply:
(I did see this, I just ignored it)